libutils/definitions.h

```diff
@@ -25,6 +25,14 @@
 #ifndef CFENGINE_DEFINITIONS_H
 #define CFENGINE_DEFINITIONS_H
 
+/*****************************************************************************
+ * Metric prefixes
+ *****************************************************************************/
+#define KIBIBYTE(n) n * 1024
+#define MEBIBYTE(n) n * 1024 * 1024
+#define GIBIBYTE(n) n * 1024 * 1024 * 1024
+#define TEBIBYTE(n) n * 1024 * 1024 * 1024 * 1024
```

Comment:
I would put them in parenthesis, although it probably doesn't matter in most cases.

```suggestion
#define KIBIBYTE(n) (n * 1024)
#define MEBIBYTE(n) (n * 1024 * 1024)
#define GIBIBYTE(n) (n * 1024 * 1024 * 1024)
#define TEBIBYTE(n) (n * 1024 * 1024 * 1024 * 1024)
```